### PR TITLE
[VEN-2203]: Mock token deployments for opbnbtestnet

### PR DESCRIPTION
**Pull request by GitGuru7**
Resolves VEN-2203

## Checklist

- [ ] I have updated the documentation to account for the changes in the code.
- [ ] If I added new functionality, I added tests covering it.
- [ ] If I fixed a bug, I added a test preventing this bug from silently reappearing again.
- [ ] My contribution follows [Venus contribution guidelines](docs/CONTRIBUTING.md).

### CODE REVIEW

File: helpers/deploymentConfig.ts [hunk 1]
```diff
@@ -2828,6 +2837,41 @@ export const globalConfig: NetworkConfig = {
...
+        symbol: "BTCB",
+        decimals: 18,
+        tokenAddress: "0x7Af23F9eA698E9b953D2BD70671173AaD0347f19",
+      },
+      {
+        isMock: true,
+        name: "Ethereum Token",
+        symbol: "ETH",
+        decimals: 18,
+        tokenAddress: "0x94680e003861D43C6c0cf18333972312B6956FF1",
```
[the remark this answers is not in the document]
Comment (GitGuru7) [reply helpers/deploymentConfig.ts]: [a15c8ea](https://github.com/VenusProtocol/isolated-pools/pull/329/commits/a15c8ead5ce554a71c1fddae068eb83fd29596b8)

File: helpers/deploymentConfig.ts [hunk 2]
```diff
@@ -159,6 +161,13 @@ const preconfiguredAddresses = {
     GovernorBravo: ETHEREUM_MULTISIG,
     AccessControlManager: governanceEthereum.AccessControlManager.address,
   },
+  opbnbtestnet: {
+    VTreasury: "0x3370915301E8a6A6baAe6f461af703e2498409F3",
+    NormalTimelock: OPBNBTESTNET_MULTISIG,
+    FastTrackTimelock: OPBNBTESTNET_MULTISIG,
+    CriticalTimelock: OPBNBTESTNET_MULTISIG,
+    AccessControlManager: "0x049f77F7046266d27C3bC96376f53C17Ef09c986",
```
Comment (chechu) [inline helpers/deploymentConfig.ts:169]: This address should be available on the npm package (at least in the version `1.4.0-dev.9`)

**Review by chechu — DISMISSED** [review]

File: hardhat.config.ts [hunk 3]
```diff
@@ -230,10 +243,28 @@ const config: HardhatUserConfig = {
...
+        urls: {
+          apiURL: `https://open-platform.nodereal.io/${process.env.ETHERSCAN_API_KEY}/op-bnb-testnet/contract/`,
+          browserURL: "https://testnet.opbnbscan.com/",
+        },
+      },
+      {
+        network: "opbnbmainnet",
+        chainId: 204,
+        urls: {
+          apiURL: `https://open-platform.nodereal.io/${process.env.ETHERSCAN_API_KEY}/op-bnb-testnet/contract/`,
```
Comment (chechu) [inline hardhat.config.ts:258]:
```suggestion
          apiURL: `https://open-platform.nodereal.io/${process.env.ETHERSCAN_API_KEY}/op-bnb-mainnet/contract/`,
```

**Review by chechu — DISMISSED** [review]

File: hardhat.config.ts (hunk 3 above)
Comment (GitGuru7) [reply hardhat.config.ts]: [19f26f6](https://github.com/VenusProtocol/isolated-pools/pull/329/commits/19f26f61131570829e9f4a828625caad5dda7862)

**Review by chechu — APPROVED** [review]

**Pull request merged**